values-aws.yaml

```diff
@@ -0,0 +1,112 @@
+# Copyright 2019 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+fullnameOverride: palisade-services
+
+# hosting: [local | aws | metal] the target environment to deploy to
+hosting: aws
+
+traefik:
+  # install: install the traefik ingress controller, this should be true for the first installation into the cluster as it resides in kube-system to provide a single cluster-wide ingress controller
+  install: false
+metrics-server:
+  # install: install the metrics server, this should be true for the first installation into the cluster as it resides in kube-system to provide a single cluster-wide metrics server
+  install: false
+dashboard:
+  # install: install routing through the ingress controller to a pre-installed kubernetes dashboard i.e. https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/
+  install: false
+kafka:
+  # install: install kafka streaming platform
+  install: true
+redis:
+  # install: install redis persistence and caching data storage platform
+  install: false
+redis-cluster:
+  # install: install clustered redis persistence and caching data storage platform
+  install: true
```

Comment:
Why is there two redis switches here. Surely it is a question of wanting redis (yes or no) if yes then how many nodes. If it is > 1 then it will be a clustered deployment

Reply:
There's two verieties, one of which is definitely more appropriate for a cluster, the other may be much faster to spin up for a 'local' demo or test